### PR TITLE
Implement database with specs

### PR DESCRIPTION
**Pull request by katie-truong**
Specs are in specs folder.

### CODE REVIEW

**Comment by katie-truong** [comment]
Hey @courtchao @cdl08001 @shripriyaprabhakar please review this PR. (After much struggle with Git, I decided to start a branch new repo!)

**Comment by cdl08001** [comment]
Looking at this now.

File: db/databaseGenerator.js [hunk 1]
```diff
@@ -57,21 +58,25 @@ const stores = [
     storeID: 1,
     storeName: 'Target',
     storeMinimumFreeShipping: 15,
+    logo: 'https://i.imgur.com/KHPqFqS.png',
```
Comment (cdl08001) [inline db/databaseGenerator.js]: I think all all logos\images need to be stored somewhere (such as AWS)

File: db/databaseGenerator.js [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+  user: 'root',
+  password: '',
+});
+
+connection.connect();
+
+// Drop database if it exists:
+connection.query('drop database if exists expressel;', (error) => {
+  if (error) throw error;
+});
```
Comment (cdl08001) [inline db/databaseGenerator.js:15]: Might be good to have some indicator that this completed successfully (console log message). Not required - just a suggestion

Comment (cdl08001) [reply db/databaseGenerator.js:15]: Same with the rest of the queries

**Comment by cdl08001** [comment]
@katie-truong looks good! Not that many suggestions on my end, just take a look at how you are referencing your images. I think we need to have them saved somewhere as opposed to a direct reference to a URL.

File: db/databaseGenerator.js (hunk 1 above)
Comment (katie-truong) [reply db/databaseGenerator.js]: I stored them on imgur. AWS may be a better option, which I would revisit later :) Thanks Cal!

**Pull request merged**